Duplicati/Library/Backend/AlternativeFTP/AlternativeFTPBackend.cs

```diff
@@ -451,6 +451,7 @@ private static Stream StringToStream(string str)
             var stream = new MemoryStream();
             var writer = new StreamWriter(stream) { AutoFlush = true };
             writer.Write(str);
+            stream.Position = 0;
             return stream;

```

Comment:
It seems strange to me that the caller is responsible for disposing the returned `Stream`, and that the `StreamWriter` is not disposed.  What are your thoughts on moving these statements to the one usage below and then removing this method?

Reply:
Good catch. I removed `StreamWriter` altogether.

Reply:
Thanks @sergethedev17.  Looks good to me.